homeassistant/components/mqtt/strings.json

```
@@ -1004,6 +1228,13 @@
         "rgbww": "[%key:component::light::entity_component::_::state_attributes::color_mode::state::rgbww%]",
         "white": "[%key:component::light::entity_component::_::state_attributes::color_mode::state::white%]"
       }
+    },
+    "target_temperature_feature": {
+      "options": {
+        "single": "Single target temperature",
+        "high_low": "High/low target temperature",
```

I checked the actual translations for `target_temp_high` and `target_temp_low` in `climate`.

Their friendly  names are
- Upper target temperature
- Lower target temperature

Therefore we should adapt all the strings in `mqtt` to use the same wording.
Starting here:

```suggestion
        "high_low": "Upper/lower target temperature",
```

---

We can change this option, but I would not recommend changing the other options as it makes it hard to see what the documented option is. So lets keep hose options in terms of high and low.

---

It is mainly the docs, but we can update those as well.
Do you want to do some suggestions?